UndertaleModTool/Scripts/Community Scripts/TouchControlsEnabler.csx

```diff
@@ -0,0 +1,156 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Linq;
+using UndertaleModLib.Util;
+
+EnsureDataLoaded();
+
+string display_name = Data?.GeneralInfo?.DisplayName?.Content.ToLower();
```

Comment:
please use camelCase for variables instead of snake_case.

Reply:
aight